config/webhook/manifests.yaml

```diff
@@ -1,12 +1,44 @@
 
 ---
-apiVersion: admissionregistration.k8s.io/v1beta1
+apiVersion: admissionregistration.k8s.io/v1
+kind: MutatingWebhookConfiguration
+metadata:
+  creationTimestamp: null
+  name: mutating-webhook-configuration
+webhooks:
+- admissionReviewVersions:
+  - v1
+  - v1beta1
+  clientConfig:
+    caBundle: Cg==
+    service:
+      name: webhook-service
+      namespace: system
+      path: /validate-network-kubesphere-io-v1alpha2-svc
+  failurePolicy: Fail
+  name: mutating.eip.network.kubesphere.io
+  rules:
+  - apiGroups:
+    - ""
+    apiVersions:
+    - v1
+    operations:
+    - CREATE
+    resources:
+    - services
+  sideEffects: NoneOnDryRun
+
```

Comment:
What is this for?

Reply:
It's generated by kubebuilder on adding `sideEffects: NoneOnDryRun`. See [code](https://github.com/amal-thundiyil/openelb/blob/4b8e1f99e8030e09374169ae649da7ec92e37ad6/api/v1alpha2/eip_types.go#L157). Otherwise, it errors out. [See #244 (comment)](https://github.com/openelb/openelb/issues/244#issuecomment-1146858445).